addons/fix-uploaded-svgs/userscript.js

```diff
@@ -1,4 +1,75 @@
 export default async function ({ addon, console }) {
+  // Fixes "x", "y", and "dominant-baseline" being ignored by Scratch, "span" causing offset
+  function toEditorSVG(element) {
+    // This iframe is needed to correct "dominant-baseline"
+    const iframe = document.createElement("iframe");
+    iframe.setAttribute("src", "about:blank");
+    document.body.append(iframe);
+
+    const svg = element.cloneNode(true);
+    iframe.contentDocument.body.appendChild(svg);
+
+    var transform, translate, translateIndex, x, y, difference;
+    var baselineAdjust, innerTextElement, innerTextSpan;
```

Comment:
Could these use `let` and `const` instead and be defined where they're first used (especially the constants)?

Reply:
The variables declared with "var" are reassigned inside the "for" loop... how should I define the variables?